src/optype.rs

```diff
@@ -373,32 +373,47 @@ pub enum OpType {
     /// See \ref PauliExpBox
     PauliExpBox,
 
+    /// See \ref PauliExpPairBox
+    PauliExpPairBox,
+
+    /// See \ref PauliExpCommutingSetBox
+    PauliExpCommutingSetBox,
+
     /// NYI
     CliffBox,
 
-    /// See \ref CustomGate
-    CustomGate,
```

Comment:
Is this not needed anymore?

Reply:
As far as I can tell CustomGate is the same as Composite I believe.